build/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ftlabs-screens-viewer",
-  "version": "1.0.10",
+  "version": "1.0.13",
```

Comment:
Yeah there were a few dead end deploys with now fixed bugs. I deleted the tags but don't like rolling back version numbers.